fem/bilininteg_diffusion.cpp

```diff
@@ -549,12 +972,12 @@ static void SmemPADiffusionApply2D(const int NE,
       MFEM_SYNC_THREAD;
       if (tidz == 0)
       {
-         MFEM_FOREACH_THREAD(d,y,D1D)
+         MFEM_FOREACH_THREAD(D,y,D1D)
```

Comment:
Using big D here is confusing - there's another D above.

Reply:
Done.